release-schema.json

```
@@ -56,6 +56,15 @@
               "minLength": 1
             }
           }
+        },
+        "eDeliveryGateway": {
+          "title": "eDelivery gateway",
+          "description": "The organization's URL for the exchange of data and documents.",
```

https://ec.europa.eu/digital-building-blocks/wikis/display/DIGITAL/eDelivery

```suggestion
          "description": "The organization's URL for the exchange of data and documents with eDelivery.",
```